spec/features/admin_can_create_a_category_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'admin' do
+  context 'with valid credentials' do
+    let(:admin){
+      User.create(first_name: "Admin",
+                  last_name: "astrator",
+                  username: "admin",
+                  email: "admin@admin.com",
+                  password: "password",
+                  password_confirmation: "password",
+                  role: 1)
+    }
+    it 'can create a categories' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.